src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/ConsoleNotes.java

```diff
@@ -67,11 +62,13 @@ static void parse(byte[] b, int len, Map<String, Object> data, boolean saveAnnot
                 break;
             }
         }
-        
+
         String line = new String(b, 0, eol, StandardCharsets.UTF_8);
-        
-        // Would be more efficient to do searches at the byte[] level, but too much bother for now,
-        // especially since there is no standard library method to do offset searches like String has.
+
+        // Would be more efficient to do searches at the byte[] level, but too much
```

Comment:
I would increase max. line width (from 80) to 120. What is the recommendation by GitHub for correct display?

Reply:
In GitHub up to 140 characters are visible per line.